frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeSizeExpandedField.tsx

```diff
@@ -2,19 +2,16 @@ import * as React from 'react';
 import { FormGroup, Grid, NumberInput, ValidatedOptions } from '@patternfly/react-core';
 import IndentSection from 'pages/projects/components/IndentSection';
 import { UpdateObjectAtPropAndValue } from 'pages/projects/types';
-import { CreatingServingRuntimeObject, ServingRuntimeResources } from '../../types';
+import { CreatingServingRuntimeObject } from '../../types';
 import { isHTMLInputElement, normalizeBetween } from 'utilities/utils';
+import { ContainerResourceAttributes, ContainerResources } from '../../../../../types';
 
 type ServingRuntimeSizeExpandedFieldProps = {
   data: CreatingServingRuntimeObject;
   setData: UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>;
 };
 
-type ResourceKeys = keyof ServingRuntimeResources;
-enum ResourceAttributes {
-  CPU = 'cpu',
-  MEMORY = 'memory',
-}
+type ResourceKeys = keyof ContainerResources;
 
 const ServingRuntimeSizeExpandedField: React.FC<ServingRuntimeSizeExpandedFieldProps> = ({
```

Comment:
This component got a significant facelift. You'll probably want to rebase onto #937 to solve this conflict.